### PR TITLE
Google SSO updates

### PR DESCRIPTION
**Pull request by johnthompson365**
Removed step1 as unneccessary
Changed order to enable API near start to prevent UI errors
Added authorised domains step
Updated screenshots

### CODE REVIEW

File: source/deployment/sso-google.md [hunk 1]
```diff
@@ -5,17 +5,25 @@ Follow these steps to configure Mattermost to use Google as a single sign-on (SS
...
 ![google_1_credentials](../../source/images/google_1_credentials.png)
 
-3 - Click the **OAuth consent screen** header and enter "Mattermost" as the **Product name shown to users**, then click **Save**.
+2 - In the **Dashboard** click the option to **ENABLE APIS AND SERVICES**.
+
+![google_enable_api](../../source/images/google_enable_api.png)
+
+Once in the API Library, scroll down to [Google+ API](https://console.developers.google.com/apis/api/plus/overview) and click **Enable** in the header. This might take a few minutes to propagate through Google's systems.
+
+3 - Leave the Google+ API menu and return to the **APIs & Services menu** and in the left hand sidebar select **Credentials**, click the **OAuth consent screen** header and enter "Mattermost" as the **Product name shown to users**, then click **Save**.
```
Comment (jasonblais) [inline source/deployment/sso-google.md]:
```suggestion
3 - Leave the Google+ API menu to return to the **APIs & Services menu**. In the left hand sidebar select **Credentials**, click the **OAuth consent screen** header and enter "Mattermost" as the **Product name shown to users**, then click **Save**.
```

File: source/deployment/sso-google.md [hunk 2]
```diff
@@ -5,17 +5,25 @@ Follow these steps to configure Mattermost to use Google as a single sign-on (SS
...
 
 ![google_2_oauth_consent_screen](../../source/images/google_2_oauth_consent_screen.png)
 
-4 - Under the **Credentials** header, click **Create credentials**, choose **OAuth client ID** from the drop-down list.
+Scroll further down and you will need to add the domain used by your Mattermost instance to a list that is authorised to access the API. For example, if it is hosted on `mattermost.yourdomain.com` you would add `yourdomain.com`.
```
Comment (jasonblais) [inline source/deployment/sso-google.md]:
```suggestion
Scroll further down to add your Mattermost instance to a list of authorized domains that can access the API. For example, if it is hosted on `mattermost.yourdomain.com` add `yourdomain.com`.
```

**Comment by johnthompson365** [comment]
Updated the changes. Is there any issue with the new screenshots not being same size as rest on page?

**Comment by jasonblais** [comment]
Thanks! `../google_authorised_domains.png` is a bit smaller compared to the other images, but otherwise it seems fine, at least here https://github.com/mattermost/docs/blob/2b54151e988ebc2eadc9c4857da7f0f0cc95018b/source/deployment/sso-google.md

Or did you feel the images were much different in size?

**Comment by johnthompson365** [comment]
Ok, as long as you are happy. Didn't know how particular we were.

**Pull request merged**